recipes/cmcrameri/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cmcrameri" %}
+{% set version = "0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c7a7533647629ec620128c4adbeb03ffc308e4ffef7dab2af7fb87176977d65
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```

Comment:
I think this recipe can be `noarch`, so please add `noarch: python`, and remove the `skip` line. With that, in the `requirements` section you should use `python >=3.6`, https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-builds.